php/class-deactivation.php

```diff
@@ -230,14 +276,23 @@ public function rest_callback( WP_REST_Request $request ) {
 			return rest_ensure_response( 418 );
 		}
 
-		$url = add_query_arg(
-			array(
-				'reason'    => sanitize_text_field( $reason ),
-				'free_text' => sanitize_textarea_field( $more ),
-			),
-			self::$cld_endpoint
+		$args = array(
+			'reason'    => sanitize_text_field( $reason ),
+			'free_text' => sanitize_textarea_field( $more ),
 		);
 
+		if ( $report ) {
+			$report = $this->upload_report();
+
+			if ( ! empty( $report['secure_url'] ) ) {
+				$args['report'] = $report['secure_url'];
+			}
+
+			$args['contact'] = $contact;
+		}
+
+		$url = add_query_arg( $args, self::$cld_endpoint );
+
 		$response = wp_remote_get( $url ); // phpcs:ignore WordPressVIPMinimum.Functions.RestrictedFunctions.wp_remote_get_wp_remote_get
```

Comment:
@pereirinha - I know this is existing code, but I think we should use `wp_safe_remote_get()` here, so we don't need the ignore.

Reply:
Thanks, @DavidCramer 

Great catch.